libfabric.spec

```diff
@@ -8,7 +8,7 @@
 
 Name: libfabric
 Version: %{major}.%{minor}.%{bugrelease}%{?prerelease:~%{prerelease}}
-Release: 2%{?dist}
+Release: 3%{?dist}
```

Comment:
Don't really need to bump this for just adding new packages IMO.

Reply:
Fair enough, although that's what we did in https://github.com/daos-stack/daos/blob/master/utils/rpms/daos.spec#L675-L676

Reply:
That one did include a [general package composition change](https://github.com/daos-stack/daos/commit/9f87976898ee9b6a16f312127ace61a6e6898f26#diff-157dea0c50d0811d0acdc68394f79827b7e57044a6b765c2f2cfead1150226efR385) although admittedly the `%changelog` did not mention it.  It probably should have.  Talk about burying the headline.  :frowning: 

That said, I'm not at all adamant about whether a change like this bumps or not, so proceed as you wish, by all means.